modules/ROOT/pages/import-template.adoc

```diff
@@ -0,0 +1,43 @@
+= Create a Mule Application Quickly from a Template
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+:imagesdir: ../assets/images
+
+The easiest way to get started in Flow Designer is to use a template.  A template is a pre-built integration that you can customize.  Go to Anypoint Exchange to see templates for the most popular integration use cases. You can open templates in read-only mode in Flow Designer and review them. If you believe that a template will help you create the Mule application that you need, you can create a new project from it, customize the project, and then test and deploy your application.
```

Comment:
"the most popular"

Reply:
I disagree. It really is the easiest way to get started in FD. It can't be the most popular because we are introducing the feature in this release.